src/app/drive/components/FileViewer/FileViewer.tsx

```diff
@@ -53,65 +55,119 @@ const FileViewer = ({ file, onClose, onDownload, downloader, show }: FileViewerP
   const isTeam = useAppSelector(sessionSelectors.isTeam);
   const userEmail: string = localStorageService.getUser()?.email || '';
 
+  const handleFileThumbnail = async (file: DriveFileData, fileBlob: Blob) => {
+    const currentThumbnail = file.thumbnails && file.thumbnails.length > 0 ? file.thumbnails[0] : null;
+    const fileObject = new File([fileBlob], file.name);
+    const fileUpload: FileToUpload = {
+      name: file.name,
+      size: file.size,
+      type: file.type,
+      content: fileObject,
+      parentFolderId: file.folderId,
+    };
+
+    const thumbnailGenerated = await getThumbnailFrom(fileUpload);
+
+    if (
+      thumbnailGenerated &&
```

Comment:
Please extract these comparisons in a boolean variable whose name explains why is the check being done, in two months we will be very grateful for doing that

Reply:
Some questions that could arise and can help you to choose a naming for the variables:
- Why do I check `.file` property nullability?
- Why do I check `.type` property nullability?

Reply:
Finally here I have removed the unnecessary comparisons leaving only the thumbnailGenerated.file. 
It was not possible to extract it into an explicitly named boolean because typescript cannot infer the type inside the conditional.